proposals/src/mappings/mappingHandlers.ts

```diff
@@ -189,65 +228,91 @@ export async function handleVote(
     contract,
     proposalNumber,
     proposalOpen,
-    true
+    new Date(header.time)
   );
   if (!proposal) {
+    logger.error(
+      `----- ${contract} > ${proposalNumber} ==> Failed during vote`
+    );
     return;
   }
 
   // Create and save wallet voted proposal object with timestamp.
   await (
     await ProposalVote.create({
-      id: `${contract}:${proposalNumber}:${sender}`,
+      id: `${proposal.id}:${sender}`,
```

Comment:
i don't think this new key is unique?

Reply:
proposal.id is set to that string i removed (referring to the graphql entity model), it's not the id from the on-chain proposal object

Reply:
ohh sick nice sorry should have read more closely :)